optimum/intel/openvino/modeling_base.py

```diff
@@ -439,7 +439,7 @@ def from_pretrained(
 
             ov_files = _find_files_matching_pattern(
                 model_dir,
-                pattern=r"(.*)?openvino(.*)?\_model.xml",
+                pattern=r"(.*)?openvino(.*)?\_model.xml$",
```

Comment:
good catch !

Reply:
As a bit of extra context: The `HfApi().snapshot_download` with local_files creates a `.cache` folder with every file in the remote repository appended with `.metadata`. The resulting `openvino_model.xml.metadata` was also matched by the pattern.